PYTHON3.md

```diff
@@ -37,32 +41,52 @@ change the variables you want and use them with
 summon make -f Makefile.frankfurt ENV_FILE=.env.mine build serve
 ```
 
-## Docker build
+:book: You need some external ressource to run the service, see [External ressources dependencies](#external-ressources-dependencies)
+
+## External Ressources Dependencies
+
+To run the service locally you need to have access to the following external ressources:
+
+- Postgresql database `pg-geodata-replica.bgdi.ch`
+- S3 bucket `service-mf-chsdi3-grid-geojsons-dev-swisstopo` (NOTE: this bucket is only required by some of the Identity endpoints)
+
+### Postgresql Port Forwarding
+
+You can use the ssh port forwarding feature to have access to `pg-geodata-replica.bgdi.ch` by using the jump host:
 
 ```bash
-    make -f Makefile.frankfurt dockerbuild
+ssh ssh0a.prod.bgdi.ch -L 5432:pg-geodata-replica.bgdi.ch:5432
 ```
 
-## Docker run
+Then set the `DBHOST` environment variable to `localhost` (you can do this in your own environment file e.g. `.env.mine` and run the make file as follow: `summon make -f Makefile.frankfurt ENV_FILE=.env.mine serve`)
+
+### S3 Vector Bucket Access
+
+To have access to the S3 bucket, you can either set your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variable. **:warning: When using those variables make sure that the command are not saved into history file !**
+
+Alternatively if you are using `zsh` you can use the `aws` plugin (see [oh-my-zsh aws plugin](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/aws)) with the following command:
```

Comment:
Hmm, I'm not sure it's good idea to disclose all this information in a public repo. We can either move it to infra-kubernetes to a readme in this services folder or think about a general place where we document how to develop locally (I think we have to do this anyway somewhere for data integration).